src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -140,6 +143,16 @@ public void mouseReleased(final MouseEvent e) {
         if (terr != null) {
           notifyTerritorySelected(terr, md);
         }
+        if(e.getButton() == 4 || e.getButton() == 5){
+          if (is4Pressed && is5Pressed) {
+            lastActive = e.getButton() == 4 ? 5 : 4;
+          } else {
+            lastActive = -1;
+          }
+          is4Pressed = e.getButton() == 4 ? false : is4Pressed;
+          is5Pressed = e.getButton() == 5 ? false : is5Pressed;
+          return;
```

Comment:
We want to prevent the gui from interpreting button 4 and 5 as clicks wich would cause the gui to open menus while scrolling